config-linux.md

```diff
@@ -742,6 +742,8 @@ If `intelRdt` is not set, the runtime MUST NOT manipulate any `resctrl` pseudo-f
 The following parameters can be specified for the container:
 
 * **`closID`** *(string, OPTIONAL)* - specifies the identity for RDT Class of Service (CLOS).
+  As a special case, value `/` means that the container should be assigned to the default CLOS (the
```

Comment:
Should bus use `SHOULD` (or `MUST`, whatever applies), to follow [rfc2119](https://datatracker.ietf.org/doc/html/rfc2119) /  [rfc8174](https://datatracker.ietf.org/doc/html/rfc8174) ?

Reply:
Thanks. Changed to MUST